config/_default/menus.yaml

```diff
@@ -11,139 +11,150 @@
-    weight: 150
+    weight: 8
     identifer: WP5
     parent: About
   - name: WP6 - Project Management
     pageRef: workpackages/06_Project_Management
-    weight: 160
+    weight: 9
     identifer: WP6
     parent: About
 
-  - name: News
-    weight: 20
-    pageRef: news
-
+  # Services tab
   - name: Services
-    weight: 60
+    weight: 2
   - name: RSQkit
-    weight: 10
+    weight: 1
     pageRef: services/RSQkit
     parent: Services
   - name: SMP
-    weight: 20
+    weight: 2
     pageRef: services/SMP
     parent: Services
 
-
+  # Training tab
   - name: Training
-    weight: 80
+    weight: 3
   - name: Training Events
-    weight: 10
+    weight: 1
     pageRef: training/events
     parent: Training
   - name: Training Materials
-    weight: 20
+    weight: 2
     pageRef: training/material
     parent: Training
   - name: Recognition
-    weight: 30
+    weight: 3
     pageRef: training/recognition
     parent: Training
 
+  # Resources tab
   - name: Resources
-    weight: 100
+    weight: 4
   - name: Deliverables and Milestones
-    weight: 10
+    weight: 1
     pageRef: resources/deliverables_milestones
     parent: Resources
   - name: Presentations
-    weight: 20
+    weight: 2
     pageRef: resources/presentations
     parent: Resources
   - name: Visual Identity
-    weight: 30
+    weight: 3
     pageRef: resources/visual_identity
     parent: Resources
   - name: Zotero and Zenodo
-    weight: 40
+    weight: 4
     pageRef: resources/zetoro_zenodo
     parent: Resources
 
+  # Network tab
   - name: Network
-    weight: 120
+    weight: 5
   - name: Communities
-    weight: 10
+    weight: 1
     pageRef: network/communities
     parent: Network
   - name: Competence Centres
-    weight: 20
+    weight: 2
     pageRef: network/competence_centres
     parent: Network
   
+  # Participate tab
   - name: Participate
-    weight: 140
+    weight: 6
   - name: How to Join
     pageRef: participate/how_to_join
-    weight: 10
+    weight: 1
     parent: Participate
 
+  # News tab
+  - name: News
+    weight: 7
+    pageRef: news
+
+
 footer:
   - name: GitHub
     url: https://github.com/EVERSE-ResearchSoftware
     pre: github
-    weight: 100
-  - name: Mastodon
-    url: https://mastodon.instance/@username
-    pre: mastodon
     weight: 110
+  # - name: Mastodon
+  #   url: https://mastodon.instance/@username
+  #   pre: mastodon
+  #   weight: 110
   - name: Email
-    url: mailto:everse@everse.software
+    url: mailto:everse-request@lists.certh.gr
     pre: email
-    weight: 120
+    weight: 100
   - name: LinkedIn
-    url: https://linkedin.com/in/username
+    url: https://www.linkedin.com/company/eosc-everse
     pre: linkedin
-    weight: 140
+    weight: 120
   - name: Zenodo
-    url: https://zenodo.org/
+    url: https://zenodo.org/communities/everse
     pre: zenodo
-    weight: 150
+    weight: 140
+  - name: Twitter
+    url: https://twitter.com/eosc_everse
+    pre: twitter
```

Comment:
Should change to the `x-twitter` icon (😭)

Reply:
I updated the icon.